### ISSUE TITLE
Incorrect urth-core-function syntax in meetup-streaming

### ISSUE THREAD

**Issue by parente** [jupyter-incubator/dashboards#186]
Code cell #4 needs to adapt to the new arg syntax like so:

```
    <urth-core-function auto
        id="set_topic_filter"
        ref="set_topic_filter"
        arg-value="{{topic_filter}}">
```

**Comment by parente** [comment]
@nitind can you make this simple change and send a PR when you get a minute?

**Comment by nitind** [comment]
Opened PR#187.

**Comment by parente** [comment]
Fixed in PR #187.
